### ISSUE TITLE
Margin and styling issues 

### ISSUE THREAD

**Issue by Mariam-mohamed110** [Sigma-Labs-XYZ/explain-ai#53]
At the moment the breadcrumbs and title for topic card have a few styling issues:
- The breadcrumbs should be inlined with the topic card component, in a desktop setting 
- The second tag in a mobile setting has too much margin making it look inconsistent
- The title for the topic card component is not bold and standing out 

<img width="336" alt="Screenshot 2023-03-03 at 10 17 15 am" src="https://user-images.githubusercontent.com/104569563/222702375-cdc03957-3780-4e08-b086-7a9a30c22ea3.png">
<img width="336" alt="Screenshot 2023-03-03 at 10 21 04 am" src="https://user-images.githubusercontent.com/104569563/222702383-30a53fe1-51ce-4396-8fc5-eced6baa3065.png">

**Comment by DomVinyard** [comment]
Absolutely, let's make this a general Ticket to better resemble the Topic Page mockup - including keeping all of the elements in line

<img width="584" alt="image" src="https://user-images.githubusercontent.com/1271197/222717034-97c79498-e0a4-4d21-bfcf-fba11c3c419e.png">

### Completion criteria

- Match the mockup design as closely as possible
